_posts/2017-10-05-apprenticeship-at-8thlight.md

```diff
@@ -0,0 +1,213 @@
+> We enjoy working with people who are empathetic, who are passionate about teaching and learning, and who value the work they do. If this approach to software development resonates with you, we'd love to talk.
+
+[crafter-application]: https://8thlight.com/job-opportunities/london-crafter/ 
+[jakes-tictactoe]: http://jakescruggs.blogspot.de/2004/05/my-apprenticeship-may-2004.html
+
+The actual online process is a two-step process. First the applicant gets to know someone from 8th Light and gets a task to solve. Normally its Tic Tac Toe. The programming language is not fix. Later the applicant sends in the code and waits for feedback. There will be a code review process going on internally. I assume, if the code is not devastating a second call will be set up where the code is being discussed. The important thing here is to find out how well an applicant takes feedback. 8th Light wants to find out the ability to learn and to incorporate feedback into the applicant’s work. This will trigger a second round of programming for the applicant, where they apply the learnings and sends back the code again.
+If all went well the online part is done and the following step is going to be a face to face invitation. 
+There are some hurdles and many options for both sides to drop out of the process. I assume the people who arrive in the office for the face to face sessions will have invested a considerable amount of time and know why they are doing it all.
+
+And here is the important part, in my opinion. In order to have the applicants know the “why” and invest (so much), 8th Light must have been doing a good job in selling that they can and will deliver the “why”. They must be recognised in the ways that they attract the people with the right mindset.
+Part of how we learned about them was that they are always active at community events. They talk about their processes, the apprenticeship program and our craft. They are for one promoting these ideas of working in software development for their own good as well as for improving the industry as such. And they have well-known and outstanding people who work for them and do this promotion. It’s the crafters who talk about it, not a marketing department or alike. They work hard to be authentic.
+
+## The application, part 2, in person
+
+The applicants who get invited into the office pair with the 8th Lighters all day. They pair with two different people. One in the morning, then its lunch time and the socialising part takes place. Some say this is the most important part. And in the afternoon pairing continues with a different pair.
+As far as I understood that is the entire application process. After this all the people who had been involved sit together and decide if that person gets invited. And if so, the question is the “position” for which the applicant fits.
+
+## Overview
+
+There are actually only two “positions” for which 8th Light hires: 1) student apprentices or 2) resident apprentices.
```

Comment:
We were in the process of changing this during your visit. https://8thlight.com/job-opportunities. All positions require an apprenticeship. Student apprenticeships are very different in that they're unpaid / free for the apprentice, and they are not necessarily a path to employment. More of an internship, or a free code school.